### PR TITLE
std.random unittest docs

### PR DESCRIPTION
**Pull request by burner**
some examples could be moved unittests

### CODE REVIEW

File: std/random.d [hunk 1]
```diff
@@ -1044,6 +1037,17 @@ alias Xorshift160 = XorshiftEngine!(uint, 160, 2,  1,  4);  /// ditto
 alias Xorshift192 = XorshiftEngine!(uint, 192, 2,  1,  4);  /// ditto
 alias Xorshift    = Xorshift128;                            /// ditto
 
+///
+unittest{
```
Comment (quickfur) [inline std/random.d]: Please follow Phobos coding style here.

Comment (burner) [inline std/random.d]: done

**Comment by quickfur** [comment]
LGTM

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

**Pull request merged**